### PR TITLE
Added team-Documentation to maintainer's guide

### PR DESCRIPTION
**Pull request by sventiffe**

### CODE REVIEW

**Review by aiuto — APPROVED** [review]
LGTM, but since you are making floriographygoth a contact, they should approve too.

**Review by jin — APPROVED** [review]

**Review by aiuto — APPROVED** [review]

**Comment by aiuto** [comment]
ping.  I can merge this, but like I said before, floriographygoth should approve since it makes them the contact.

**Pull request closed without merging**
